app/models/pdc_metadata/related_object.rb

```diff
@@ -5,12 +5,25 @@ module PDCMetadata
   # let(:related_identifier_type) { "arXiv" }
   # let(:relation_type) { "IsCitedBy" }
   class RelatedObject
-    attr_accessor :related_identifier, :related_identifier_type, :relation_type
+    attr_accessor :related_identifier, :related_identifier_type, :relation_type, :errors
 
     def initialize(related_identifier:, related_identifier_type:, relation_type:)
       @related_identifier = related_identifier
       @related_identifier_type = related_identifier_type
       @relation_type = relation_type
+      @errors = []
+    end
+
+    def valid?
+      valid = related_identifier.present? && valid_related_identifier_type? && valid_relation_type?
+      return valid if valid
+      if related_identifier.blank?
+        errors << "Related identifier is missing"
+      else
+        errors << "Related Identifier Type is missing or invalid for #{related_identifier}" unless valid_related_identifier_type?
+        errors << "Relationship Type is missing or invalid for #{related_identifier}" unless valid_relation_type?
```

Comment:
```suggestion
        @errors << "Related identifier is missing"
      else
        @errors << "Related Identifier Type is missing or invalid for #{related_identifier}" unless valid_related_identifier_type?
        @errors << "Relationship Type is missing or invalid for #{related_identifier}" unless valid_relation_type?
```

I might be missing something, but wouldn't plain `errors` just be a local variable in the method?